django_saml2_auth/views.py

```diff
@@ -181,11 +182,12 @@ def acs(r):
     # BEGIN TESORIO CHANGES
     # saml_client = _get_saml_client(get_current_domain(r))
     saml_metadata_conf_url = r.session.get('saml_metadata_conf_url')
-    if not saml_metadata_conf_url:
-        logger.warning("No saml_metadata_conf_url found", extra={"session": dict(r.session)})
+    saml_metadata_conf_raw = r.session.get('saml_metadata_conf_raw')
+    if not saml_metadata_conf_url and not saml_metadata_conf_raw:
+        logger.warning("No saml_metadata_conf found", extra={"session": dict(r.session)})
```

Comment:
I think we can demote this to `info` @Isaben

Reply:
Done